File: src/scripts/model/_DataFileMetaData.ts
export class DataFileMetaData {

    id: number

    name: string

    createdBy: string

    dateModified: string

    constructor(
        id: number,
        name: string,
        createdBy: string,
        dateModified: string
    ) {
        this.id = id;
        this.name = name;
        this.createdBy = createdBy;
        this.dateModified = dateModified;
    }
}

File: ASPNetCoreAssignment/ASPNetCoreAssignment.Models/DataFileMetaData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ASPNetCoreAssignment.Models
{
    public class DataFileMetaData
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string CreatedBy { get; set; }
        public string DateModified { get; set; }
        public string Type { get; set; }
    }
}
